test/test_monitoring.py

```diff
@@ -496,32 +494,30 @@ def test_exhaust(self):
         self.listener.results.clear()
         tuple(cursor)
         results = self.listener.results
-        started = results['started'][0]
-        succeeded = results['succeeded'][0]
         self.assertEqual(0, len(results['failed']))
```

Comment:
Note that in the interest of making the test portable, we cannot assert anything about the length of `succeeded` and `started`. The reason being, `OP_REPLY` doesn't respect the `batchSize` and simply returns all 6 results in its response to the first `getMore`.

Reply:
Interesting! I wonder if that's expected behavior or an OP_REPLY bug in pymongo or the server.